editor/block-mover/style.scss

```diff
@@ -3,7 +3,6 @@
 	top: 10px;
 	left: 0;
 	padding: 0 14px 20px 0; // handles hover area
-	z-index: z-index( '.editor-block-mover' );
```

Comment:
@aduth That sounds like a good idea, and I think that at least lowering the z-index for the corresponding SASS style sheet would probably be the safer, less invasive alternative, since removing this line here ultimately would have the same effect of reducing the z-index in the corresponding SASS style sheet, just a much more lowering effect.

Forgive me if this is the wrong place to ask this, but while I have your attention on this issue, I'd like to ask a bigger question: 

*What is the plan for introducing the drag and drop ability for the blocks to re-order and position themselves by dragging them around, and perhaps add new blocks by dragging them in from a list of available block choices?*

For a form-based editor WP plugin that I am building for a client, so that users can build forms by dragging around form elements and add in new form elements by dragging them in to the main form 'tree', I've worked considerably with [react-sortable-tree](https://github.com/fritz-c/react-sortable-tree), even creating the feature myself for giving its API the ability to drag in new external nodes, from [this merged PR](https://github.com/fritz-c/react-sortable-tree/pull/113). React-sortable-tree's dragging ability is built solely on its bedrock of [React-dnd](http://react-dnd.github.io/react-dnd/).

I've enjoyed the abstracted, focus-on-the-data-not-the-DOM approach of React Dnd, and I am curious what you think of the possible application of React Dnd (or possibly even react-sortable-tree) is for Gutenberg.